test/helper.rb

```diff
@@ -58,8 +58,11 @@ module TimeoutEveryTestCase
   class TestTookTooLong < Timeout::Error
   end
 
-  def capture_exceptions(*)
-    ::Timeout.timeout(RUBY_ENGINE == 'ruby' ? 60 : 120, TestTookTooLong) { super }
+  def time_it
+    t0 = Minitest.clock_time
+    ::Timeout.timeout(RUBY_ENGINE == 'ruby' ? 60 : 120, TestTookTooLong) { yield }
+  ensure
+    self.time = Minitest.clock_time - t0
```

Comment:
TIL :-) 👍 

minitest source code links for anyone else curious:

* https://github.com/seattlerb/minitest/blob/32d49db55d80b8479237898f07d161bb52ef905c/lib/minitest.rb#L264-L275
* https://github.com/seattlerb/minitest/blob/e6bc4485730403faff6966c1671cf5de72b2d233/lib/minitest/test.rb#L89-L110